src/classes/CW_AssociateAudienceRewardHelper.cls

```diff
@@ -0,0 +1,236 @@
+/**
+* File Name   :    CW_AssociateAudienceRewardHelper.cls
+* Description :    B-114504 - Create Audience Group Schedule Reward records
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 28/05/2014 Alex Panka	  Intial version 
+*/
+//Cristina P: 25/06/14: Changing the class from "with sharing" to "without sharing" 
+//so the user can create related records while cost/brief is in approval
+public without sharing class CW_AssociateAudienceRewardHelper 
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.